static/sass/_patterns_strip_slanted.scss

```diff
@@ -63,7 +63,26 @@
   .p-strip-slanted--snapcraft {
     @extend %vf-strip;
     @extend %vf-strip-slanted;
-    background-image: url("#{$assets-path}a3c29307-hero-bg-snapcraft-1--nowhite.svg");
+    // background-image: url("#{$assets-path}a3c29307-hero-bg-snapcraft-1--nowhite.svg");
+    background-image:
+    linear-gradient(
```

Comment:
Is it possible to ident the three `linear-gradient` so it is more clear they are for this `background-image`?

Reply:
Will, try to find a better solution, but the scss linter is complaining.